code/modules/events/disease_outbreak.dm

```diff
@@ -9,13 +9,28 @@
 		D = new virus_type()
 	else
 		var/datum/disease/advance/A = new /datum/disease/advance
+		// Try generating a random spreading virus. If that fails just use the last generated one
+		for(var/i = 0; i < 100; i++)
```

Comment:
I would use the `for(var/i in 1 to 100)` syntax, as it's faster than this traditional for loop syntax.

Reply:
![image](https://user-images.githubusercontent.com/25063394/88420444-d2207e00-cdde-11ea-86e1-ebac79833012.png)

Reply:
Oh wew interesting. I'll change it